docs/documentation/siddhi-4.0.md

```diff
@@ -46,6 +46,208 @@ by a semicolon `( ; )` as shown in the below syntax.
         ;
 ```
 
+**Performance**
+
+To evaluate the performance of an application, you can enable the statistics of a Siddhi application to be published. This is done via the `@app` annotation that can be added to a Siddhi application as shown in the following example.
+
+```sql
+@app:statistics(reporter = 'console')
+```
+The following elements are configured with this annotation.
+
+|Annotation| Description| Default Value|
+| ------------- |-------------|-------------|
+|`reporter`|The type of the reporter for statistics. Possible values are as follows:<br/> `console`<br/> `jmx`<br/> `jdbc`<br/> If `jdbc` is specoified, the statistics are stored in the `WSO2_METRICS_DB` datasource (which is configured in the `<SP_HOME>/conf/<PROFILE>/deployment.yaml` file. |`console`|
+|`interval`|The time interval (in seconds) at  which the statistics for the Siddhi application are reported.|`60`|
+
+e.g., the following is a Siddhi application that includes the `@app` annotation to report performance statistics.
+
+```sql
+@App:name('TestMetrics')
+
+@app:statistics(reporter = 'console')
+
+define stream TestStream (message string);
+
+define trigger FiveMinTriggerStream at every 5 sec;
+
+@info(name='query1')
+from FiveMinTriggerStream
+select "Hi " as message
+insert into TestSream;
+
+@info(name='logQuery')
+from TestSream#log("Message:")
+insert into TempSream;
+
+```
+
+Statistics are reported for this Siddhi application as shown in the extract below.
+
+<details>
+  <summary>Click to view the extract</summary>
+11/26/17 8:01:20 PM ============================================================
+  -- Gauges ----------------------------------------------------------------------
```

Comment:
The metrics under gauges are provided only in carbon environment not for @App: statistics annotation